2022/04/07 06:09:16 [error] 32#32: *1 open() "/var/www/html/favicon.ico" failed (2: No such file or directory), client: 172.27.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2022/04/07 12:03:10 [error] 37#37: *5 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: localhost:9003 (through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.22.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.22.0.3:9000", host: "localhost"
2022/04/07 12:03:10 [error] 37#37: *5 open() "/var/www/html/favicon.ico" failed (2: No such file or directory), client: 172.22.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2022/08/01 12:00:35 [error] 33#33: *1 upstream timed out (110: Operation timed out) while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.3:9000", host: "localhost"
2022/08/01 12:00:35 [error] 33#33: *1 open() "/var/www/html/favicon.ico" failed (2: No such file or directory), client: 172.50.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2022/09/23 09:02:50 [error] 24#24: *49 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.3:9000", host: "localhost"
2022/09/23 09:02:51 [error] 24#24: *49 open() "/var/www/html/favicon.ico" failed (2: No such file or directory), client: 172.50.0.1, server: localhost, request: "GET /favicon.ico HTTP/1.1", host: "localhost", referrer: "http://localhost/"
2022/09/23 09:03:16 [error] 24#24: *49 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.3:9000", host: "localhost"
2022/09/23 09:03:16 [error] 24#24: *49 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.3:9000", host: "localhost"
2022/09/23 09:03:19 [error] 24#24: *49 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.3:9000", host: "localhost"
2022/09/23 09:04:34 [error] 24#24: *49 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.3:9000", host: "localhost"
2022/09/23 09:06:05 [error] 24#24: *49 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.3:9000", host: "localhost"
2022/09/23 09:06:06 [error] 24#24: *49 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.3:9000", host: "localhost"
2022/09/23 09:09:29 [error] 23#23: *59 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.3:9000", host: "localhost"
2022/09/23 09:11:51 [error] 23#23: *62 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.3:9000", host: "localhost"
2022/09/23 09:14:11 [error] 26#26: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 09:14:12 [error] 26#26: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 09:14:15 [error] 26#26: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 09:42:38 [error] 26#26: *6 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 12:06:47 [error] 26#26: *22 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 12:16:45 [error] 25#25: *8 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 12:18:55 [error] 25#25: *2 upstream timed out (110: Operation timed out) while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 12:24:40 [error] 26#26: *14 recv() failed (104: Connection reset by peer) while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 12:39:13 [error] 25#25: *18 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 13:04:14 [error] 25#25: *21 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 13:08:21 [error] 26#26: *23 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 13:08:22 [error] 26#26: *23 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 13:13:26 [error] 26#26: *27 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 13:14:46 [error] 26#26: *27 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 13:16:02 [error] 26#26: *27 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 13:19:36 [error] 25#25: *1 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 13:22:43 [error] 25#25: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 13:22:46 [error] 25#25: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 13:26:34 [error] 25#25: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 13:26:41 [error] 25#25: *9 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 13:30:12 [error] 25#25: *12 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 13:30:16 [error] 25#25: *12 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 13:30:17 [error] 25#25: *12 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 13:30:58 [error] 24#24: *2 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 13:48:07 [error] 25#25: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
2022/09/23 13:48:09 [error] 25#25: *4 FastCGI sent in stderr: "PHP message: Xdebug: [Step Debug] Could not connect to debugging client. Tried: 172.50.0.1:9005 (from REMOTE_ADDR HTTP header), host.docker.internal:9005 (fallback through xdebug.client_host/xdebug.client_port) :-(" while reading response header from upstream, client: 172.50.0.1, server: localhost, request: "GET / HTTP/1.1", upstream: "fastcgi://172.50.0.2:9000", host: "localhost"
